### PR TITLE
Toggle cleanup

### PR DESCRIPTION
**Pull request by samrocksc**
Updated toggles so that they can be used with ID's properly.  Updated index.js to reflect the changes as well.

### CODE REVIEW

**Comment by samrocksc** [comment]
I think this covers all of the id carriers

File: src/checkbox.js [hunk 1]
```diff
@@ -32,10 +36,12 @@ class Checkbox extends React.Component {
       id
     } = this.props;
 
-    const classes = classnames(baseClasses, {
+    const labelClasses = classnames(labelBaseClasses, {
       'mdl-js-ripple-effect': ripple
     }, className);
 
+    const inputClasses = classnames(inputBaseClasses);
```
Comment (phated) [inline src/checkbox.js]: `className` property should be applied here and no on `labelClasses`

File: src/checkbox.js [hunk 2]
```diff
@@ -32,10 +36,12 @@ class Checkbox extends React.Component {
       id
     } = this.props;
 
-    const classes = classnames(baseClasses, {
+    const labelClasses = classnames(labelBaseClasses, {
       'mdl-js-ripple-effect': ripple
     }, className);
```
Comment (phated) [inline src/checkbox.js]: `className` should not be applied here

File: src/radio.js [hunk 3]
```diff
@@ -32,10 +37,12 @@ class Radio extends React.Component {
       id
     } = this.props;
 
-    const classes = classnames(baseClasses, {
+    const labelClasses = classnames(labelBaseClasses, {
       'mdl-js-ripple-effect': ripple
     }, className);
```
Comment (phated) [inline src/radio.js]: `className` should not be applied here

File: src/radio.js [hunk 4]
```diff
@@ -32,10 +37,12 @@ class Radio extends React.Component {
...
 
+    const inputClasses = classnames(inputBaseClasses)
```
Comment (phated) [inline src/radio.js]: `className` property should be applied here and no on `labelClasses`

File: src/switch.js [hunk 5]
```diff
@@ -32,10 +36,12 @@ class Switch extends React.Component {
       id
     } = this.props;
 
-    const classes = classnames(baseClasses, {
+    const labelClasses = classnames(labelBaseClasses, {
       'mdl-js-ripple-effect': ripple
     }, className);
```
Comment (phated) [inline src/switch.js]: `className` should not be applied here.

File: src/switch.js [hunk 6]
```diff
@@ -32,10 +36,12 @@ class Switch extends React.Component {
...
 
+    const inputClasses = classnames(inputBaseClasses);
```
Comment (phated) [inline src/switch.js]: `className` property should be applied here and no on `labelClasses`

**Comment by samrocksc** [comment]
Made Changes, fixed placement of className.

**Comment by phated** [comment]
@samrocksc looks like the `switch.js` file was altered to be tab indented

**Comment by samrocksc** [comment]
fixed the tab spacing, just had to pull in to 2-spacing standards

**Comment by samrocksc** [comment]
Think we're good here

**Comment by phated** [comment]
@samrocksc nope, still wrong.  you have to look at the diff you produce here on github to see where you are unnecessarily changing things

**Comment by samrocksc** [comment]
leaving nothing but pertinent changes

**Pull request merged**